Gemfile

```diff
@@ -70,7 +75,7 @@ end
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
-# json genarator
-gem 'active_model_serializers'
+# ActiveModel::Serializer implementation and Rails hooks.
+gem 'active_model_serializers', '~> 0.10.0'
```

Comment:
I specified the lowest version in `Gemfile.lock`. I am worried about inhibiting version up by writing this.